scripts/onboarding/aks/onboarding-msi-bicep-syslog/existingClusterParam.json

```diff
@@ -56,6 +56,18 @@
           "<existingOrnew-tag-name2>": "<existingOrnew-tag-value2>",
           "<existingOrnew-tag-nameN>": "<existingOrnew-tag-valueN>"
         }
+      },
+      "dataCollectionInterval":  {
+        "value" : "1m"
+      },
+      "namespaceFilteringModeForDataCollection": {
+        "value": "Off"
+      },
+      "namespacesForDataCollection": {
+        "value": [ "kube-system", "gatekeeper-system", "azure-arc" ]
+      },
+      "enableContainerLogV2": {
+        "value": true
```

Comment:
Dont merge this PR until the current agent rollout complete as the current rolling agent version 3.1.10 has the changes for the containerlogv2 dcr changes.